pymc3/examples/discreteuniform_test.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+from pymc3 import *
+
+
+def run_MCMC(lower, upper, obs, draws):
```

Comment:
We're using pytest for testing, you can check out some other tests but at the minimum this funciton name needs to start with `test_`.